packages/crew_profile_base.rb

```diff
@@ -0,0 +1,91 @@
+require 'package'
+
+class Crew_profile_base < Package
+  description 'Crew-profile-base sets up Chromebrew\'s environment capabilities.'
+  homepage 'https://github.com/chromebrew/crew-profile-base'
+  version '0.0.1'
+  license 'GPL-3+'
+  compatibility 'all'
+  source_url 'file:///dev/null'
+  source_sha256 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'
```

Comment:
Why not use a release?  https://github.com/chromebrew/crew-profile-base/archive/refs/tags/0.0.1.tar.gz